Gemfile

```diff
@@ -46,6 +46,7 @@ gem 'devise-jwt', '~> 0.9.0'
 
 gem 'aws-sdk-s3', '~> 1.111'
 
-gem 'cors', '~> 1.0'
 
 gem "rspec-rails", "~> 5.1", :groups => [:development, :test]
+
+gem "rack-cors", "~> 1.1"
```

Comment:
'rack-cors' seems to be the gem people are using, not 'cors'
https://medium.com/ruby-daily/a-devise-jwt-tutorial-for-authenticating-users-in-ruby-on-rails-ca214898318e
https://tergell.medium.com/rails-devise-jwt-tutorial-b5d5b03d9040

Reply:
yeah i saw that yesterday when i was googling something related